packages/backend/src/config/app.ts

```diff
@@ -187,7 +187,8 @@ export const Config = {
     'INCOMING_PAYMENT_EXPIRY_MAX_MS',
     2592000000
   ), // 30 days
-  enableSpspPaymentPointers: envBool('ENABLE_SPSP_PAYMENT_POINTERS', true)
+  enableSpspPaymentPointers: envBool('ENABLE_SPSP_PAYMENT_POINTERS', true),
+  maxPaymentRetryAttempts: envInt('MAX_STATE_ATTEMPTS', 5)
```

Comment:
Let's call this "max outgoing payment retry attempts" in both the env variable and the config variable.